project/.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
You'll need to wrap them in quotes, because in YAML 3.10 would be loaded as a float `3.1`.